index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile_website</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="wrapper">
+    <div class="wrapper-1">
+      <header>
+        <div class="logo"><a href="">My Logo</a></div>
+        <nav>
+          <a href="">
+            <img id="menu" src="./images/menu.png" alt="menu">
+          </a>
```

Comment:
- [ ] You did an excellent job on the design but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template as much as possible. Please reduce the size of the hamburger button to match the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185658062-67a15334-81cf-4509-aeb8-f10b79ee9268.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/185658099-8d3311cc-c1c2-451d-a288-e7d2c70e97fb.png)